src/components/Text/index.ts

```diff
@@ -1,5 +1,3 @@
 import DefaultText from './DefaultText'
 
-import styles from './styles'
-
-export { DefaultText, styles }
+export { DefaultText }
```

Comment:
ITextProps is not exported again!
This also raises a small naming issue, we have DefaultText(which sounds ugly) and we have its props interface with a name ITextProps. Inconsistent.

Reply:
fixed